src/main/java/dp/api/dataset/RetryStrategy.java

```diff
@@ -0,0 +1,35 @@
+package dp.api.dataset;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.ServiceUnavailableRetryStrategy;
+import org.apache.http.protocol.HttpContext;
+
+/**
+ * Custom implementation of ServiceUnavailableRetryStrategy to retry any HTTP 5xx responses.
+ */
+public class RetryStrategy implements ServiceUnavailableRetryStrategy {
+
+    private final int maxRetries;
+    private final long retryInterval;
```

Comment:
Its not easy to see what time measurement retryInterval is.

Reply:
Added a suffix onto the variable name `retryIntervalMs`